### ISSUE TITLE
Clicking on Whale button 

### ISSUE THREAD

**Issue by mchiang0610** [kitematic/kitematic#525]
Clicking on Whale button (Docker CLI) without previously having terminal app started would create 2 instances of the terminal. 

Reproduce procedure: 

1. Quit Terminal.app 
2. Click on Whale button

**Comment by mhelmetag** [comment]
I'm pretty certain this bug is caused by [this line](http://github.com/mhelmetag/kitematic/blob/master/resources/terminal#L58). I feel like that's opening another tab.

Maybe we could check if an instance of Terminal is running and only run this line when it is...

I may look into fixing this but... anyone feel free to try presenting a solution.

**Comment by thaJeztah** [comment]
I wonder if that line was added on purpose, because an existing terminal may already be running another process, and may not have the correct environment variables setup for Docker Toolbox; opening a fresh tab works around those possibilities

**Comment by mhelmetag** [comment]
Right. I would assume so (didn't look at the commit message). That seems correct.

I think that, if one were so inclined, you would rewrite this a bit to check if terminal was already open or not and... if it was, open a new tab with the right setup and... if not, open only one window.
